app/uk/gov/hmrc/cgtpropertydisposalsfrontend/models/LocalDateUtils.scala

```diff
@@ -39,6 +39,14 @@ object LocalDateUtils {
 
   def today(): LocalDate = LocalDate.now(clock)
 
+  def dateFromString(
```

Comment:
you can just use `LocalDate.of` if you need to construct a `LocalDate`. It avoids people wondering whether we've got the right string format for the date when constructing dates.

Reply:
Done.